src/pages/patientView/PatientViewPage.tsx

```diff
@@ -669,42 +676,36 @@ export default class PatientViewPage extends React.Component<
                                     this.toggleGenePanelModal,
                                     this.genePanelModal.isOpen
                                 )}
-                            </span>
-                            {this.patientViewPageStore
-                                .hasMutationalSignatureData.result === true && (
-                                <LoadingIndicator
-                                    isLoading={
-                                        this.patientViewPageStore
-                                            .mutationalSignatureData
-                                            .isPending &&
-                                        this.patientViewPageStore
-                                            .mutationalSignatureMetaData
-                                            .isPending
-                                    }
-                                />
-                            )}
-
-                            {this.patientViewPageStore
-                                .hasMutationalSignatureData.result === true &&
-                                this.patientViewPageStore
-                                    .clinicalDataGroupedBySample.isComplete &&
-                                this.patientViewPageStore
-                                    .mutationalSignatureData.isComplete &&
-                                this.patientViewPageStore
-                                    .mutationalSignatureMetaData.isComplete && (
-                                    <SignificantMutationalSignatures
-                                        data={
-                                            this.patientViewPageStore
-                                                .mutationalSignatureData.result
-                                        }
-                                        metadata={
+                                {this.patientViewPageStore
+                                    .hasMutationalSignatureData.result && (
```

Comment:
it would probably make more sense to just put the check again isPending after && and then set isLoading=true.